src/DetailView/DetailView.js

```diff
@@ -1,53 +1,73 @@
+import { keyboard } from "@testing-library/user-event/dist/keyboard"
 import React, { Component } from "react"
 import "./DetailView.css"
+import backButton from "../assets/back-button.png"
 
 class DetailView extends Component {
   constructor(props) {
     super(props)
     this.movieId = this.props.movieId
     this.closeMovie = this.props.closeMovie
-    this.state = {
-    }
+    this.state = {}
   }
 
   componentDidMount() {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.movieId}`)
       .then(response => response.json())
       .then(movieObject => {
         const data = movieObject.movie
+        const releaseDateObject = new Date(Date.parse(data.release_date))
+        const dateData = `${releaseDateObject.toLocaleString("default", { month: "long" })} ${releaseDateObject.getDate()}, ${releaseDateObject.getFullYear()}`
+        const dateYear = releaseDateObject.getFullYear()
 
         this.setState({
           averageRating: data.average_rating,
           backdropPath: data.backdrop_path,
-          budget: data.budget,
-          genres: data.genres,
+          budget: data.budget.toLocaleString(),
+          genres: data.genres.join(' '),
           id: data.id,
           overview: data.overview,
           posterPath: data.poster_path,
-          releaseDate: data.release_date,
-          revenue: data.revenue,
+          releaseDate: releaseDateObject,
+          dateData: dateData,
+          dateYear: dateYear,
+          revenue: data.revenue.toLocaleString(),
           runtime: data.runtime,
-          tagline: data.tagline,
           title: data.title
         })
       })
+
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.movieId}/videos`)
+      .then(response => response.json())
+      .then(videoObject => {
+        const data = videoObject.videos[0]
+        const isEmbedAvailable = data === [] ? false : true
+        if (isEmbedAvailable) {
+          this.setState({
+            videoUrl: data.key
+          })
+        }
+      })
   }
 
   render() {
     return <>
```

Comment:
This is cool! I didn't realize you could get around having to nest everything in another `<div>` by using the empty brackets like this.

Reply:
Yeah, React supports HTML fragments that let you basically make a wrapper for the purposes of the `return` statement. I'm not 100% clear if there are best practices surrounding the use of these, but for now I find they're a lot more convenient (and I _feel_ like they tend to be more semantic/readable) then using a `div` tag.